test/setup_mock.py

```diff
@@ -47,24 +52,75 @@ def setup_mock_wrapper(self, mock_3parclient, mock_etcd, mock_fileutil,
             mock_protocol_connector = mock_fc_connector
 
         with mock.patch.object(hpecommon.HPE3PARCommon, '_create_client') \
-            as mock_create_client, \
-            mock.patch.object(mgr.VolumeManager, '_get_etcd_util') \
-            as mock_get_etcd_util, \
-            mock.patch.object(mgr.VolumeManager, '_get_connector') \
+                as mock_create_client, \
+                mock.patch.object(mgr.VolumeManager, '_get_etcd_util') \
+                as mock_get_etcd_util, \
+                mock.patch.object(orchestrator.Orchestrator,

```

Comment:
I think ideally orchestrator should not be mocked as this will leave it untested via UT. It would be better to mock oslo_config instead.

As an alternate, we can continue to mock orchestrator and in order to exercise its functionality, separate unit test(s) can be implemented outside the UT framework that can just load the orchestrator and invoke its functionality.

Reply:
Will be done in phase 2 of this implementation.